### PR TITLE
Adding angular variables and b-tag sorting

### PR DESCRIPTION
**Pull request by milosdjordjevic**
This is to add the additional (angular) variables intended to use for the ttH signal extraction. Also, a module is added (sort_3l) to save the index of the jet with the highest b-tag (for 8 TeV comparison).

### CODE REVIEW

File: CMGTools/TTHAnalysis/python/tools/angular_vars.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+			  "pTl_DR_l_b_loose1", "pTl_DR_l_b_medium1", "pTl_DR_l_b_tight1", 
+			  "pTl_DR_l_b_loose2", "pTl_DR_l_b_medium2", "pTl_DR_l_b_tight2",  
+			  "L_pt" ]
+    def listBranches(self):
+        return self.branches[:]	
+    def __call__(self,event):
+        # make python lists as Collection does not support indexing in slices
+	leps = [l for l in Collection(event,"LepGood","nLepGood",4)]
+        jets = [j for j in Collection(event,"Jet","nJet25",8)]
+	bloose  = [j for j in jets if j.btagCSV > 0.423]
```
Comment (peruzzim) [inline CMGTools/TTHAnalysis/python/tools/angular_vars.py]: @milosdjordjevic, could you please check these working points and update them to the latest recommended ones if needed?

**Comment by milosdjordjevic** [comment]
Please find the updated working points to the latest recommendation

File: CMGTools/TTHAnalysis/python/tools/angular_vars.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              ret["DR_l_b_medium1"] = min([deltaR(n,leps[0]) for n in bmedium]) if nlep >= 1 else 0;
+	if nbtight >= 1:
+	   if leps[0].relIso03 < 0.1:
+              ret["DR_l_b_tight1"] = min([deltaR(p,leps[0]) for p in btight]) if nlep >= 1 else 0;
+	      
+	#DR_l_b_2:
+
+	if nbloose >= 1:
+	   if leps[1].relIso03 < 0.1:
+              ret["DR_l_b_loose2"] = min([deltaR(q,leps[1]) for q in bloose]) if nlep >= 1 else 0;
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/tools/angular_vars.py]: this and below should be  `if nlep >= 2` rather than `if nlep >= 1` since you access leps[1]

**Comment by gpetruc** [comment]
After some discussion with @peruzzim  yesterday: we'll fix the merge conflicts when we'll also adapt the code to run on the output of the re-cleaner.

**Comment by gpetruc** [comment]
Closing, development has moved on 76X and this feature is already there.

**Pull request closed without merging**